### DISCUSSION TITLE
Tuya THB1 & TH05 Aliexpress

### DISCUSSION THREAD

**Discussion by ncodee** [pvvx/ATC_MiThermometer#423] [Q&A] [answered]
Has anyone tested the TH05 and THB1 Bluetooth Temperature and humidity sensors that can be readily found on Aliexpress?

They look a lot like that Xiaomi models with the screens, and suspect use the same microcontrollers.

Link: https://a.aliexpress.com/_ExKqZvL

Thanks

**Comment by pvvx** [comment, marked as answer]
https://pvvx.github.io/TH-05/
https://pvvx.github.io/THB2/

Look into https://pvvx.github.io

**Reply by ncodee** [reply]
Thanks!
